sections/semantics-grouping-content.include

```diff
@@ -1248,10 +1252,13 @@
     </xmp>
   </div>
 
-  Each term within a <a>term-description group</a> must be represented by a single <{dt}> element. The descriptions within a <a>term-description group</a> are alternatives. Each description must be represented by a single <{dd}> element.
+  Each term within a <a>term-description group</a> must be represented by a single <{dt}> element.
```

Comment:
Which means that we have an HTML validity error in many algorithm switches, because they use multiple `dt`s as alternatives to be addressed by a single following `dd`.

I'll raise a separate issue on that. EIther we are doing the wrong thing, or the spec is overly prescriptive and unhelpful, but I don't know yet...

Reply:
Hmm. Reading again, I see that it does allow multiple terms in the same group.